indico/modules/events/sessions/templates/management/_sessions_list.html

```diff
@@ -0,0 +1,97 @@
+                                {%- trans %}Poster{% endtrans -%}
+                            {% else %}
+                                {%- trans %}Standard{% endtrans -%}
+                            {% endif -%}
+                        </td>
+                        <td class="i-table">
+                            <div class="group">
+                                <span class="js-color-switch">
+                                    <a class="i-button icon-palette switch-trigger"
+                                       style="background: #{{ session.background_color }};
+                                              color: #{{ session.text_color }} !important"
+                                       data-method="PATCH" data-href="{{ url_for('.change_colors', session) }}"
+                                       data-colors="{{ default_colors | tojson | forceescape }}">
+                                    </a>
+                                </span>
+                            </div>
+                        </td>
+                        <td class="i-table">
+                            <div class="group right">
+                                <a class="icon-edit highlight edit-session action-icon"
+                                   title="{% trans %}Edit this session{% endtrans %}"
+                                   data-title="{% trans title=session.title %}Edit session {{ title }}{% endtrans %}"
+                                   data-href="{{ url_for('.modify_session', session) }}"></a>
+                                <a class="icon-users highlight session-persons-list"
+                                   data-title="{% trans title=session.title %}List of session persons{% endtrans %}"
+                                   data-href="#"></a>
+                                <a class="icon-time highlight action-icon"
+                                   title="{% trans %}Go to this session timetable{% endtrans %}"
+                                   data-href="#"
+                                   data-method="POST"></a>
+                                <a class="icon-remove highlight js-remove-session action-icon"
+                                   title="{% trans %}Remove this session{% endtrans %}"
+                                   data-method="DELETE"
+                                   data-href="{{ url_for('.delete_session', session)}}"
+                                   data-confirm="{% trans title=session.title %}Do you really want to delete session '{{ title }}'?{% endtrans %}"></a>
+                            </div>
+                        </td>
+                    </tr>
+                    {% for block in session.blocks -%}
+                        <tr class="i-table session-blocks-row" style="display: none;">
+                            <td class="i-table" colspan="3"></td>
+                            <td class="i-table">{{- block.title or 'Default empty block' -}}</td>
```

Comment:
Does this default title make sense? I'd rather show `-` or even an empty column if there's no block title